recipes/pyretis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
```

Comment:
The version 1.00 will be released soon on pypt. I tried to link the gitlab directly but I didn't manage to make it lint... If you have time to assist me I can try to make a new pull request for that, but I am afraid it is getting dispersive.